### PR TITLE
modal background and general style changes

### PR DESCRIPTION
**Pull request by avsssai**
made the modal to a dark color, centered the X button. and styled the help section differently.

Its an amazing concept! will contribute more!

### CODE REVIEW

**Comment by pavankalyan-codes** [comment]
@avsssai Thanks for the contribution! could you pls take a look at merge conflicts and resolve them?

**Comment by avsssai** [comment]
done 👍

File: src/styles/styles.js [hunk 1]
```diff
@@ -1,8 +1,11 @@
 export const customStyles = {
-  backgroundStyle: {
-    backgroundColor: "black",
-    height: "100%"
+  // backgroundStyle: {
```
Comment (pavankalyan-codes) [inline src/styles/styles.js:2]: @avsssai pls remove the commented code. we have git for storing history.

**Comment by pavankalyan-codes** [comment]
@avsssai , Everything looks good to me. let me know once you remove comments - will merge. Thanks

**Pull request merged**